src/api/urls.py

```diff
@@ -135,3 +135,7 @@
         ),
     ),
 ]
+
+apiurl = [
+    url(r"^/wp-json/extensions/v3/$", include(urlpatterns)),
+]
```

Comment:
According to the docs [(URLconf)](https://docs.djangoproject.com/en/3.1/topics/http/urls/#how-django-processes-a-request) django will load the url config and look for a variable called `urlpatterns`. So one option would be to append this list to `urlpatterns` sequence. But then there is the problem, that in our ROOT_URLCONF [API-urls](https://github.com/Integreat/integreat-cms/blob/e3b7c733822b3be05a5202b9b2e1a6c605e4e681/src/backend/urls.py#L25) all urls in this file get a preceeding `api` and this violates wordpress API specification.
Do you have an idea how to solve this problem?

Reply:
@melegiul Good catch! I suggest to move the "api" prefix to the [src/api/urls.py](https://github.com/Integreat/integreat-cms/blob/563c174ac36cde9e922aa2b3a31c245e9e1da93b/src/api/urls.py) file in the following way:

```
api_urlpatterns = [
    # All existing urlpatterns here
]

urlpatterns = [
    url(r"^api/", include(api_urlpatterns)),
    url(r"^wp-json/extensions/v3/", include(api_urlpatterns)),
]
```
Apart from that, there are also a few endoints which are named differently (e.g. "sites" vs "regions"), so for this cases, we'd need a new additional rules.